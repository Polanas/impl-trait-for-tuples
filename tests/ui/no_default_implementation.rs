#[impl_trait_for_tuple::impl_for_tuples(1)]
trait Test {
	fn test() { println!("Hello"); }
}

fn main() {}
