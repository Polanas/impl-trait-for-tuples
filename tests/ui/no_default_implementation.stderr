error: Not supported by full-automatic tuple implementation. Use semi-automatic tuple implementation for more control of the implementation.
 --> $DIR/no_default_implementation.rs:3:12
  |
3 |     fn test() { println!("Hello"); }
  |               ^^^^^^^^^^^^^^^^^^^^^^
